src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/DisconnectParams.cs

```diff
@@ -15,5 +15,11 @@ public class DisconnectParams
         /// or a virtual file representing an object in a database.         
         /// </summary>
         public string OwnerUri { get; set; }
+
+        /// <summary>
+        /// The type of connection we are disconnecting. If null, we will disconnect all
+        /// connections. This is null by default.  
+        /// </summary>
+        public string Type { get; set; } = null;
```

Comment:
>= null [](start = 41, length = 6)

I think the value would be null by default if you don't set it, so this seems unneeded in this case.